contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,101 @@
+
+* **It’s clearer. **Seriously, have you ever read a transcript of you speaking? Writing things down helps with intentionality as, hopefully, it makes you think through both what you’re trying to achieve and how you want to communicate it.
+
+### The problem: Meetings
+
+Online meetings are the scourge of remote work – and often a cause of tension between office-based and non-office based employees. Mics off when they should be on, screenshare chaos, [Jeffrey Toobin’s penis](https://www.wired.com/story/jeffrey-toobin-zoom-penis/) – they remain a technical nightmare and a productivity black hole. Many companies see them as a necessary evil – a way for colleagues, no matter where they’re based, to collaborate. This is wrong.
+
+Since the pandemic, the amount of time people have spent not just in online meetings, but in meetings in general, has soared. [Time spent in meetings](https://www.microsoft.com/en-us/worklab/work-trend-index/great-expectations-making-hybrid-work-work) has gone up 253%. Some Teams users [spend](https://www.microsoft.com/en-us/worklab/work-trend-index/will-ai-fix-work) close to eight hours a week, or an entire workday, in online meetings. Eight hours a week! In Teams! Madness. And if you’re a manager or executive within an organization, this is all your fault.
+
+### The solution: Kill meetings
+
+Way back in 2009, Y Combinator co-founder Paul Graham [wrote](https://www.paulgraham.com/makersschedule.html) about the difference between the “maker’s schedule” and the “manager’s schedule.” In an office setting, these two schedules (sort of) work side by side. Managers spend their days in hour-long chunks full of meetings, check-ins, brainstorms, and walking around open-plan offices. A maker’s schedule, which is basically the schedule of an engineer, is the opposite: deep work, minimal meetings, headphones, Slack. When the pandemic forced almost every office-based company in the world to go full-time remote, these two worlds smashed together.
+
+At a lot of companies, this has forced way too many people onto the “manager’s schedule.” If this is your company, burn your weekly schedule to the ground and start again. At PostHog, we’re [anti-meeting by default](https://posthog.com/blog/meetings). We have two meeting-free days a week and, when we do have meetings, they’re short and focused. This allows not just engineers, but everyone at the company, to focus on doing their best work how they see best. We communicate asynchronously via GitHub and Slack. 
+
+A lot of companies, big and small, have adopted this approach. GitLab and Automattic for starters, but also [Twist](https://async.twist.com/asynchronous-communication/), [Buffer](https://buffer.com/about), and [Zapier](https://zapier.com/jobs/culture-and-values-at-zapier) have all followed a similar blueprint to make remote work actually work for both managers and engineers.
+
+### The problem: Everyone will slack off
```

Comment:
```suggestion
### Problem #4: Lack of trust
```

I think this is a bit clearer. The problem isn't that people do slack off, it's that managers think they will. Will have to reword the below to fit a bit.